languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/disambiguation.xml

```diff
@@ -2174,14 +2174,14 @@ Copyright © 2013 Markus Brenneis, Daniel Naber, Jan Schreiber
 
     <rule name="auf-... und zumachen etc." id="VERBPRÄFIX_HYPHEN">
         <!--Wir müssen los- und fortrennen.-->
-        <!--TO TEST: Das Los- und Fortrennen ist keine gute Idee.-->
+        <!--Das Los- und Fortrennen ist keine gute Idee.-->
         <pattern>
             <marker>
                 <token regexp="yes">(ge|fort|bei?|a[bn]|au[fs]|e(in|nt|r)|hin(ein|aus)?|her(ein|aus)?|los|mi(t|ss)|vor(wärts)?|weg|zu(rück)?|r(aus|ein)|ver|zer|um|über|unter|durch)-</token>
             </marker>
             <token regexp="yes">und|oder|&amp;|\/|sowie|noch|\+|plus|beziehungsweise|bzw</token>
             <token min="0" regexp="yes">wieder|dann|\.</token>
-            <token postag_regexp="yes" postag="VER.*" regexp="yes">(ge|fort|bei?|a[bn]|au[fs]|e(in|nt|r)|hin(ein|aus)?|her(ein|aus)?|los|mi(t|ss)|vor|weg|zu(rück)?|r(aus|ein)|ver|zer|um|über|unter|durch).+</token>
+            <token postag_regexp="yes" postag="VER.*|SUB.*NEU.*" regexp="yes">(ge|fort|bei?|a[bn]|au[fs]|e(in|nt|r)|hin(ein|aus)?|her(ein|aus)?|los|mi(t|ss)|vor|weg|zu(rück)?|r(aus|ein)|ver|zer|um|über|unter|durch).+</token>
```

Comment:
sieht alles ok aus. werden feminine Nomen schon gefunden "Be- und Entladung", oder so?

Reply:
Guter Punkt, danke!
**Please note**: durch den Marker wird auch nur der erste Teil ignoriert (expected behaviour), d.h. wenn im zweiten Teil z.b. 1 tipo drin ist, wird es dennoch als Fehler markiert)